github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: <reviewers here>
+
+JIRA: <link to applicable JIRA ticket>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
+- [ ] Passes tests
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other TODOs>
+
+### Feedback:
+_none so far_
```

Comment:
Instead of a separate "Feedback" section, could we phrase the feedback as todo items and add them to the "Todos" section?